features/search.feature

```diff
@@ -0,0 +1,23 @@
+Feature:
+  As an employee I should be able to search for things I am looking for
+    and have results displayed if any found
+
+  Background:
+    Given I have an account
+    When I login
+
+  Scenario: I visit the employees page and search
+    And I navigate to "/employees"
+    And I search for "frank"
+    Then I should see "Frank"
+
+  Scenario: I visit the meetings page and search
+    And I navigate to "/meetings"
+    And I search for "synergy"
+    Then I should see "Synergy"
+
+  # Cannot be used until one search bar on rooms page.
+  # Scenario: I visit the rooms page and search
```

Comment:
We can give each of the search bars an ID and then in the step we can look them up based on their ID tag.